### ISSUE TITLE
qTox doesn't compile

### ISSUE THREAD

**Issue by suhr** [qTox/qTox#636]
```
g++ -c -pipe -O2 -std=c++0x -Wall -W -D_REENTRANT -fPIE -DGIT_VERSION="\"aa98222e11f5b0d53155d2f0eb24566b8fe218b1\"" -DLOG_TO_FILE -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_XML_LIB -DQT_SQL_LIB -DQT_CORE_LIB -I/usr/lib64/qt5/mkspecs/linux-g++ -I. -Ilibs/include -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtOpenGL -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtNetwork -isystem /usr/include/qt5/QtXml -isystem /usr/include/qt5/QtSql -isystem /usr/include/qt5/QtCore -I. -I. -o generalform.o src/widget/form/settings/generalform.cpp
src/widget/form/settings/generalform.cpp: В функции-члене «void GeneralForm::onTimestampSelected(int)»:
src/widget/form/settings/generalform.cpp:175:59: ошибка: no match for «operator[]» (operand types are «QStringList» and «<unresolved overloaded function type>»)
     Settings::getInstance().setTimestampFormat(timeFormats[index]);
                                                           ^
src/widget/form/settings/generalform.cpp:175:59: замечание: candidates are:
In file included from /usr/include/qt5/QtCore/qvariant.h:39:0,
                 from /usr/include/qt5/QtCore/QVariant:1,
                 from ./ui_generalsettings.h:12,
                 from src/widget/form/settings/generalform.cpp:17:
/usr/include/qt5/QtCore/qlist.h:478:17: замечание: const T& QList<T>::operator[](int) const [with T = QString]
 inline const T &QList<T>::operator[](int i) const
                 ^
/usr/include/qt5/QtCore/qlist.h:478:17: замечание:   no known conversion for argument 1 from «<unresolved overloaded function type>» to «int»
/usr/include/qt5/QtCore/qlist.h:482:11: замечание: T& QList<T>::operator[](int) [with T = QString]
 inline T &QList<T>::operator[](int i)
           ^
/usr/include/qt5/QtCore/qlist.h:482:11: замечание:   no known conversion for argument 1 from «<unresolved overloaded function type>» to «int»
```

(sorry for a few russian words, locale changing not want to work properly)

**Comment by tux3** [comment]
Fixed by 85aca2ca4004854bfc0c2dc762b37d5dfac80dbe

**Comment by agilob** [comment]
tpyo. Sorry, I could build it when I was working on the code, then I pushed and realized it doesn't build after make clean.
